pkg/webhook/server.go

```diff
@@ -158,13 +161,16 @@ func (s *Server) Start(stop <-chan struct{}) error {
 		return err
 	}
 
+	log.Info("webhook server is serving at ", "host", s.Host, "port", s.Port)
```

Comment:
Usually `msg` should be a complete sentence.
e.g. `log.Info("webhook server is serving", "host", s.Host, "port", s.Port)`

Ref: https://github.com/kubernetes-sigs/controller-runtime/blob/master/TMP-LOGGING.md

Reply:
done